engines/lure/detection.cpp

```diff
@@ -73,7 +91,11 @@ static const LureGameDescription gameDescriptions[] = {
 			Common::EN_ANY,
 			Common::kPlatformDOS,
 			ADGF_NO_FLAGS,
+                        #ifdef USE_TTS
```

Comment:
The formatting is off in a lot of places - mixed spaces and tabs.

Reply:
It should be fixed. Thanks!